common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationActivity.java

```diff
@@ -51,11 +51,4 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         }
         setFragment(mFragment);
     }
-
```

Comment:
do we also need to remove this in "CurrentTaskAuthorizationActivity,java"

Reply:
Yes, thanks for the catch.